src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -384,11 +388,67 @@ public Position GetIndicatorPosition(int index)
             return new Position(indicatorList[index].Position.X + container.PositionX, indicatorList[index].Position.Y + container.PositionY);
         }
 
+        /// <summary>
+        /// Minimum value.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override double AccessibilityGetMinimum()
+        {
+            return 0.0;
+        }
+
+        /// <summary>
+        /// Current value.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override double AccessibilityGetCurrent()
+        {
+            return (double)SelectedIndex;
+        }
+
+        /// <summary>
+        /// Maximum value.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override double AccessibilityGetMaximum()
+        {
+            return (double)IndicatorCount;
+        }
+
+        /// <summary>
+        /// Current value.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override bool AccessibilitySetCurrent(double value)
+        {
+            int f = (int)value;
+
+            if (f >= 0 && f <= IndicatorCount)
+            {
+                SelectedIndex = f;
+                return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Minimum increment.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override double AccessibilityGetMinimumIncrement()
+        {
+            return 1.0;
+        }
+
         /// <inheritdoc/>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override void OnInitialize()
         {
             base.OnInitialize();
+            SetAccessibilityConstructor(Role.ScrollBar, AccessibilityInterface.Value);
```

Comment:
I'm not sure whether there's any proper role for `Pagination` class in ATSPI role.
But, as you know, there's `Scrollbar` class in NUI. And `Pagination` is similar to CSS Pagination.
https://www.w3schools.com/css/tryit.asp?filename=trycss_ex_pagination

Reply:
We have tried to mimic the approach applied in case of elm_index widget. In EFL this widget when combined with "pagecontrol" style behaves and looks identically as Pagination hence as there is no dedicated "pagination" role in the set of ATSPI roles we decided to use the "scrollbar" role combined with attribute "pagecontrolbyvalue". Using a combination of role and attribute gives us the freedom of  distinction on screen-reader side between Scrollbar widget and Pagination widget cases.

Reply:
Okay, I see. When we find the proper role of ATSPI in the future, we can update it. :)